dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/package-info.java

```diff
@@ -11,4 +11,5 @@
  *
  */
 @com.ibm.websphere.ras.annotation.TraceOptions(traceGroup = "MCP")
-package io.openliberty.mcp.internal;
\ No newline at end of file
+package io.openliberty.mcp.internal;
+import com.ibm.websphere.ras.annotation.TraceOptions;
```

Comment:
Undo the changes in this file (you don't need the import as you're using the fully qualified name on the annotation anyway).